demisto_sdk/commands/common/tools.py

```diff
@@ -2022,7 +2022,7 @@ def get_all_incident_and_indicator_fields_from_id_set(id_set_file, entity_type):
 def item_type_to_content_items_header(item_type):
     converter = {
         "incidenttype": "incidentType",
-        "indicatortype": "indicatorType",
+        "reputation": "indicatorType",
```

Comment:
I'd leave a comment here (as well as in the other places) explaining in case someone searches for indicatortype in the futrue, but if you're in a hurry to merge, it's possible to do in a separate PR (to avoid retriggering CI with the new commit)

Reply:
We'll do it in a separate PR.